_posts/blog/2017-02-23-security-performance-implications.md

```diff
@@ -0,0 +1,158 @@
+
+The purpose of this two part series was to measure the performance impact of
+various security configurations on a cluster running Apache Accumulo’s
+continuous ingest suite. The tests were performed using Amazon Web
+Services (AWS), Hortonworks Data Platform 2.4 and Accumulo 1.7. Each of
+the five different security settings in Accumulo 1.7 was tested
+including: no security, SSL, Kerberos Auth, Kerberos Auth-Int, and
+Kerberos Auth-Conf.  KDC was MIT.
+
+## Intro
+
+Part 1 was run with just continuous ingest.  Based on the test results, there was a 
+measurable performance impact as each additional security configuration was 
+put in place. 
+
+## Methodology
+
+We ran 5 tests, one for each security configuration, using the
+continuous ingest testing suite that ships with Accumulo. This testing
+suite generates random graph data and inserts it into Accumulo, creating
```

Comment:
I would move the description of ContinuousIngest into the intro -- we can expand on this one, nod to it being a "standard" (e.g. HBase's variant of CI in IntegrationTestBigLinkedList)